packages/legacy/src/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -385,99 +365,34 @@ export const AddEditProviderModal: React.FunctionComponent<IAddEditProviderModal
                   />
                 ) : null}
                 {fields?.fingerprint ? (
-                  <>
-                    {!isCertificateQueryEnabled ? (
+                  <ValidatedTextInput
+                  field={fields.fingerprint}
+                  isRequired
+                  fieldId="fingerprint"
+                  formGroupProps={{
+                    labelIcon: (
+                      <Popover
+                        bodyContent={
+                          <div>
+                            If your vSphere environment uses untrusted, self-signed certificates to authenticate connections,
+                            you must specify the <code>thumbprint</code> of the vCenter Server.
```

Comment:
```suggestion
                            The provider currently requires the SHA-1 fingerprint of the vCenter Server's
                            TLS certificate in all circumstances.  vSphere calls this the server's <code>thumbprint</code>.
```

I see in the vmware docs where the thumbprint can be skipped if the server's cert can be verified with a root cert, but I don't think that is currently supported in the backend.  I suggest this version as a current state compromise.

Reply:
:-)
I'm ok with the sujested text, updated.

@sjd78 please re-review